lua/neosolarized.lua

```diff
@@ -407,7 +407,7 @@ function M.setup(opts)
 	Group.new("WhichKeyDesc", colors.cyan, colors.base02)
 
 	-- group names with an ampersand throw an error until they gain support in 0.8.0
-	if fn.has("nvim-0.8.0") then
+	if fn.has("nvim-0.8.0") ~= 0 then
```

Comment:
Not sure what's going on here. These functions are used exactly like they are in the docs (admittedly I'm currently reading .0.9.1 docs). Care to provide the error messages you get without these changes?

Reply:
The error was: "Invalid character in group name". I don't have the exact log because I upgraded Neovim due to some changes in the Treesitter plugin. I'm not an expert, but from what I could observe, the function fn.has("some version") returns 0 if Neovim is below the specified version and returns 1 if it's above. When I was using version 0.7.2, it returned 0 at this point, so it would look something like this: if 0 then <do something>.
Different from Python and other languages, 0 is considered true in Lua, so basically, the two if statements that shouldn't be executed were executed.
Thank you for your response, and sorry for my English